### DISCUSSION TITLE
Youtube Download Button

### DISCUSSION THREAD

**Discussion by mhmmdrdhiansyah** [subhra74/xdm#1097] [Q&A]
how to make download video appear when we stream video on youtube, i already install extension on firefox, i user linux mint latest version

**Comment by JuanCarlosM1** [comment]
In the latest versions of XDM, video downloads are not done in the browser extension, they are done in XDM itself
![Captura de Tela (72)](https://github.com/subhra74/xdm/assets/64020519/5b87f57b-cbde-4134-81de-8769af348ab8)
![Captura de Tela (76)](https://github.com/subhra74/xdm/assets/64020519/3f9e3d91-4755-4f7b-859e-ff370935f652)
![Captura de Tela (73)](https://github.com/subhra74/xdm/assets/64020519/4e25f999-fb25-417e-abc5-51f14bc598a1)
![Captura de Tela (75)](https://github.com/subhra74/xdm/assets/64020519/01723db1-f662-4aa6-b360-a1c46ed778a0)

**Reply by mhmmdrdhiansyah** [reply]
i use linux so i think your solutio not work after i tried that

**Comment by rustedshader** [comment]
https://github.com/subhra74/xdm/assets/69749260/2b2f88b9-d9a3-4205-8db9-60de2820cf9f

I also use linux and it's not downloading youtube videos

**Reply by ghost** [reply]
same problem here for linux. Did you find a way? I tried youtube-dl, that didn't work out as well
